src/terminal/terminal_handlers.c

```diff
@@ -1373,3 +1377,22 @@ int guac_terminal_ctrl_func(guac_terminal* term, unsigned char c) {
 
 }
 
+/**
```

Comment:
`guac_terminal_apc()` is correctly documented at the header level. There shouldn't be a duplicate Doxygen comment attached to the function body which provides additional information. The comment should either be moved within the function, or the comment should be deleted and the information instead added to the existing header-level comment.